### DISCUSSION TITLE
ValueError: min() arg is an empty sequence

### DISCUSSION THREAD

**Discussion by teukufaiz** [nikosT/Gisola#45] [General]
Dear @nikosT 
I installed and successfully ran the test benchmark. but when i try to run a FDSNWS-event using default configuration from host : http://eida.gein.noa.gr such as :
```
./gisola.py --event noa2021abuxr
```
it successfully get the data, but it gave me this error :
```
2024-04-28 05:49:16,688 ERROR Fatal Error occurred
Traceback (most recent call last):
  File "/home/Gisola/src/./gisola.py", line 194, in <module>
    inventory.getInventory()
  File "/home/Gisola/src/config.py", line 181, in inner
    func(*args, **kwargs)
  File "/home/Gisola/src/inventory.py", line 38, in getInventory
    mindist=min([rule[0] for rule in config.distRules])
ValueError: min() arg is an empty sequence
2024-04-28 05:49:16,688 INFO Moving to next event, if any
```
Even when I try to use XML file (Download FDSNWS-events in QuakeML using ObsPy), it also show the same error.

Am I missing something?

Thanks a lot

Best wishes,
Faiz

**Comment by nikosT** [comment]
Dear @teukufaiz,

Could you also share the configuration file? Probably, the `Distance Rules` in https://github.com/nikosT/Gisola/blob/main/src/config.yaml#L107 does not cover the size of the earthquake. I suggest you to use the default configuration file: https://github.com/nikosT/Gisola/blob/main/src/config.yaml

**Reply by teukufaiz** [reply]
Dear @nikosT,

I am currently using the default configuration file as suggested at https://github.com/nikosT/Gisola/blob/main/src/config.yaml. Unfortunately, I am still encountering the same error, even with the default configuration settings. Could there be another potential cause for this issue ?

the Distance Rules in config.yaml :
```
Distance:
    - [4.0, 5.0, [10, 250, ['BH','HH']]]
    - [5.1, 5.5, [40, 300, ['BH','HH']]]
    - [5.1, 5.5, [20, 100, ['HN','EN']]]
    - [5.6, 6.0, [50, 450, ['BH','HH']]]
    - [5.6, 6.0, [50, 150, ['HN','EN']]]
    - [6.1, 9.0, [100, 700, ['BH','HH']]]
```
config.yaml file :
https://drive.google.com/file/d/1LK06HIuxRiWM-0c3nm7rIMzbxVeLQdtg/view?usp=sharing

Thank you for your help.

Best wishes,
Faiz

**Comment by nikosT** [comment]
Dear @teukufaiz,
It seems that the event you tried is out of the ML range 4-9, since the event is only 1 ML: http://bbnet.gein.noa.gr/Events/2021/01/noa2021abuxr_info.html

You could try a bigger event, e.g.: http://bbnet.gein.noa.gr/Events/2024/03/noa2024gfjlq_info.html
